src/ngx_http_lua_log_ringbuf.c

```diff
@@ -0,0 +1,201 @@
+
+#ifndef DDEBUG
+#define DDEBUG 0
+#endif
+#include "ddebug.h"
+
+
+#include "ngx_http_lua_common.h"
+#include "ngx_http_lua_log_ringbuf.h"
+
+
+typedef struct {
+    unsigned    len:28;
+    unsigned    log_level:4;
+} ngx_http_lua_log_ringbuf_header_t;
+
+
+enum {
+    HEADER_LEN = sizeof(ngx_http_lua_log_ringbuf_header_t)
+};
+
+
+static void * ngx_http_lua_log_ringbuf_next_header(
```

Comment:
We should remove the space after `*`. This issue was found by the `ngx-releng` tool, BTW.